### PR TITLE
- bug fix: make compatible with github.com/rocketlaunchr/mysql-go

### PR DESCRIPTION
**Pull request by rocketlaunchr-cto**
@propersam Can you test this.

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -12,12 +12,33 @@ cloud.google.com/go/datastore v1.0.0/go.mod h1:LXYbyblFSglQ5pkeyhO+Qmw7ukd3C+pD7
 cloud.google.com/go/pubsub v1.0.1/go.mod h1:R0Gpsv3s54REJCy4fxDixWD93lHJMoZTyQ2kNxGRt3I=
```
Comment (propersam) [inline go.sum:12]: Is this file suppose to be part of the commit??

**Review by propersam — COMMENTED** [review]
To test this new fix, I updated the sql package in my dbq-blog repo to use mysql-go instead of standard sql and all sql actions work fine. (I pushed the update to a diffrnt branch in the repo [mysql-go branch](https://github.com/propersam/dbq-blog/tree/mysql-go-branch) ).

There is no way to update the unittest for this fix in dbq because we use sql-mock as sql.

File: go.sum (hunk 1 above)
Comment (pjebs) [reply go.sum:12]: yes

**Comment by pjebs** [comment]
just do a basic test. no need for unit tests.

**Review by propersam — APPROVED** [review]
I have tested the branch with my [dbq-blog repo](https://github.com/propersam/dbq-blog/tree/mysql-go-branch) and it works ok.

**Pull request merged**